ui/README.md

```diff
@@ -0,0 +1,36 @@
+# Feathr UI
+
+This is Feathr web ui frontend code
+
+## Build and run locally
+
+Make sure you have the latest node LTS installed
+
+Run the following commands in your terminal
+
+```bash
+cd ui
+npm install
```

Comment:
I'm seeing this error when running npm start:
Compiled with problems:X

ERROR

Must use import to load ES Module:
...

Could you add a note here saying: 
node -v should display a version > 16.0 ? (12.x does not work for me).